python/ray/data/_internal/stats.py

```diff
@@ -141,6 +153,46 @@ def __init__(self, max_stats=1000):
         self.max_stats = max_stats
         self.fifo_queue = []
 
+        # Ray Data dashboard metrics
+        # Everything is a gauge because we need to reset all of
+        # a dataset's metrics to 0 after each finishes execution.
+        tags_keys = ("dataset",)
+        self.bytes_spilled = Gauge(
+            DataMetric.BYTES_SPILLED.value,
+            description="Bytes spilled by dataset operators",
```

Comment:
This description isn't really accurate for the object store-based metrics, is it? Isn't it for the whole cluster?

Reply:
This is the one that uses `get_object_locations`, so it's for the dataset